### PR TITLE
Removing PyAirtable dependency (#1259)

### LINKED ISSUE #1259: 🐈 Task: Ersilia still trying to use Airtable while deciding how to fetch a model
### Summary

We need to get rid of PyAirtable as a hard dependency, however a major workflow within how ersilia fetches models still ends up using the AirtableInterface within ersilia. This happens in this [line](https://github.com/ersilia-os/ersilia/blob/6b6c6d3f271bf4db7865107320893e9c73b44bb5/ersilia/hub/fetch/fetch.py#L225) when ersilia goes through the flow of deciding how to fetch the model . 

The simplest workaround is to change the sequence of steps from:

```Python
        do_hosted = self._decide_if_use_hosted(model_id=model_id)
        if do_hosted:
            self.logger.debug("Fetching from hosted")
            self._fetch_from_hosted(model_id=model_id)
            return
        do_dockerhub = self._decide_if_use_dockerhub(model_id=model_id)
        if do_dockerhub:
            self.logger.debug("Decided to fetch from DockerHub")
            self._fetch_from_dockerhub(model_id=model_id)
            return
```

to

```Python
        do_dockerhub = self._decide_if_use_dockerhub(model_id=model_id)
        if do_dockerhub:
            self.logger.debug("Decided to fetch from DockerHub")
            self._fetch_from_dockerhub(model_id=model_id)
            return
        do_hosted = self._decide_if_use_hosted(model_id=model_id)
        if do_hosted:
            self.logger.debug("Fetching from hosted")
            self._fetch_from_hosted(model_id=model_id)
            return
```

However since we do not have many "hosted" models, I feel we should revisit if we need this functionality entirely, and if it should be more opt-in than the default. What do you think @miquelduranfrigola?

### Objective(s)

_No response_

### Documentation

_No response_

### PR DESCRIPTION
**Pull request by daud99**
Thank you for taking your time to contribute to Ersilia, just a few checks before we proceed
- [x] Have you followed the guidelines in our [Contribution Guide](https://github.com/ersilia-os/ersilia/blob/master/CONTRIBUTING.md)
- [ ] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran tests with your changes locally?

**Description**

_Replace this line with a short description here..._

**Changes to be made**

_Replace this line with how do you intend to go about getting this done..._

**Status**

_Replace this line with what you have done so far..._

**To do**

_If this is a work in progress, Replace this line with your next steps_

Is this pull request related to any open issue? If yes, replace issueID below with the issue ID
Closes #1259

### CODE REVIEW

File: ersilia/hub/fetch/fetch.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+
+
+
```
Comment (DhanshreeA) [inline ersilia/hub/fetch/fetch.py:3]: Please we can remove all this added space.

**Review by DhanshreeA — CHANGES_REQUESTED** [review]

File: ersilia/hub/fetch/fetch.py (hunk 1 above)
Comment (daud99) [reply ersilia/hub/fetch/fetch.py:3]: I'm on it. Thanks for pointing it out @DhanshreeA

Comment (daud99) [reply ersilia/hub/fetch/fetch.py]: @DhanshreeA done

Comment (daud99) [reply ersilia/hub/fetch/fetch.py]: @DhanshreeA Can you assign me some other task as well which can be exciting to do

**Comment by Malikbadmus** [comment]
Reviewed this PR and ran several tests. LGTM!

**Review by DhanshreeA — APPROVED** [review]

**Comment by DhanshreeA** [comment]
@daud99 there's a test failing. Could you inspect and update here?

**Comment by daud99** [comment]
Closes#1291

**Comment by daud99** [comment]
@DhanshreeA Okay I'm checking it.

**Comment by daud99** [comment]
@DhanshreeA 

The test failure is happening because we've hit GitHub's hourly API request limit. The code is fine - we just need to wait for the rate limit to reset and run the test again.

The error shows this clearly:

`
403 Client Error: rate limit exceeded for url: https://api.github.com/users/ersilia-os/repos
`

**Pull request merged**